Source/EventAnalysis/KTSpectrogramCollector.hh

```diff
@@ -71,49 +65,22 @@ namespace Katydid
 
             bool Configure(const scarab::param_node* node);
 
-            double GetMinFrequency() const;
```

Comment:
`SetMinFrequency()`, `SetMaxFrequency()`, `SetMinBin()`, and `SetMaxBin()` all have non-standard behavior for setters.  You need to leave the custom setter functions and use `MEMBERFUNCTION_NOSET`.

Reply:
Done